### ISSUE TITLE
Eval is Evil

### ISSUE THREAD

**Issue by vedantterkar** [Jmchin/js-calc#1]
https://github.com/Jmchin/js-calc/blob/80517132021a0f7638d5a2099e3953112d7a8c7b/main.js#L28

Was just going through the code. Maybe there is no other way to do that. However, Be aware: https://javascriptweblog.wordpress.com/2010/04/19/how-evil-is-eval/

**Comment by Jmchin** [comment]
Yeah, no worries. I'm well aware of the dangers of eval. This was mostly an exercise in just getting something presentable, without needing to worry about any of the technical implementation required from a robust solution. There's probably libraries out there that solve the problem that I could have used, but this was a 2-hour exercise from 4 years ago when I was first teaching myself web development. I could always implement the math as a stack, but that was more trouble than it was worth for the exercise.

Thanks for the review though, Vedant!
